tasks/deploy/update.js

```diff
@@ -31,6 +31,9 @@ module.exports = function (gruntOrShipit) {
     .then(setCurrentRevision)
     .then(function () {
       shipit.emit('updated');
+    })
+    .catch(function(err) {
```

Comment:
Catching here will not make the task to fail because the error is not transmitted.